src/app/ApplicationTemplate.UWP/Views/Styles/Controls/ListView.xaml

```diff
@@ -18,52 +18,12 @@
 	-->
 
 	<Style TargetType="ListView"
+		   BasedOn="{StaticResource MaterialListViewStyle}"
 		   x:Key="DefaultListViewStyle">
 		<!-- This toolkit property overrides the padding and sets one that will make sure items are visible
 			within the safe zone. Important to support devices with round corners or notches (iPhone X, Essential Phone) -->
 		<Setter Property="toolkit:VisibleBoundsPadding.PaddingMask"
```

Comment:
While we're here, I think this PaddingMask property should not be there. It was the source of some bugs in a recent project.